azure_pipeline.yaml

```diff
@@ -308,3 +328,17 @@ stages:
               serviceConnection: ${{ deployment.service_connection }}
               terraformInitSubscription: ${{ variables.terraformInitSubscription }}
               product: ${{ variables.product }}
+        - ${{ if eq(deployment.pipeline_tests, true) }}:
```

Comment:
I think this should have a parameter for the test to run.

It's pointless running the existing test added for APIM

Reply:
Do you mean a parameter that can be used to set the type of test to run if there are multiple tests to choose from?

Reply:
yes, i.e. apim will want to run different tests. could be left for now but i.e. I wouldn't expect all components to need the same tests.

Reply:
How about what I've added [here](https://github.com/hmcts/azure-platform-terraform/pull/1231/files#diff-ac7c0ec21b006c4edccc0f5da7fd89bc8bd73dc77ee8c6ec6b117bb39747416cR323-R327)? You would set `pipeline_tests: true` and if the component is `apim` it will run whatever ends up being configured

Reply:
I think it should be a `test_pattern` parameter which should be passed to jest, and can likely be enough to enable the stage